src/js/lib/log-server/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I don't understand this change. Why this file is added now / why it was missing before? This PR doesn't seem to have any connection to this file. (This question is the only blocker from merging it to me)

Reply:
Indeed, the file is not directly related to the purpose of this PR. I added it, so I can test if the crash reporting still works.

The file existed before but was deleted by accident, as far as I can tell. Or it was deleted with a good reason that I currently don't understand. I am not very familiar with Lerna. Is it correct that this package should have its own `package.json`? Or should that configuration go somewhere else?

Reply:
Im surprised the file was not there tbh. Could you please confirm it works before merging this PR? I mean - could you confirm this server compiles and runs based on this file config?

Reply:
I was also surprised. Maybe it got lost because we have `package.json` in our `.gitignore`.

The server does run with that config.